### PR TITLE
NEWS.md: release notes for 0.20.0

### PR DESCRIPTION
**Pull request by garlick**
OK, this is kinda premature, but I wanted to see what we'd done since last release and figured I may as well open a pull request.  Force push at will.

### CODE REVIEW

**Review by grondo — APPROVED** [review]
LGTM. I assume if all looks good we can merge this in and tag tomorrow?

One comment -- the shell PMI improvement seems like it could go into the New features section since it was a large improvement using a new PMI mechanism, rather than lost and forgotten down in steerage with the Cleanups!

**Comment by garlick** [comment]
OK, I didn't know if that one was really user-visible or not, but I'm good with that change.   I'll let you do it in case you have any other tweaks you'd like to push.

**Comment by grondo** [comment]
Ok, I think I got in the change I wanted (and the typo you noticed).
This is probably ready to go in after one more proofread.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+### New features
+
+ * hwloc: add printing of num GPUs to `flux hwloc info` (#3217)
+ * resource: mark nodes down when they are stopped (#3207)
+ * broker:  allow late-joining brokers, execute rc1/rc3 on all ranks (#3168)
+ * shell/pmi: add improved PMI key exchange mechanism (#3219)
+
+### Fixes
+
+ * job-info: handle annotations race (#3196, #3208)
```
Comment (chu11) [inline NEWS.md:19]: PR #3196 is different from #3208 perhaps could be

job-manager / job-info: synchronize job priority changes from job-manager to job-info (#3208)

Comment (garlick) [reply NEWS.md]: Oh thanks, I messed that up.

I broke out 3208 again and scrunched your description down just a bit to fit on a line:
> * job-manager: communicate job priority changes manager to job-info (#3208)

Hope that's OK

**Comment by garlick** [comment]
Asan builder failed here.  Restarting.
```
ERROR: t3000-mpi-basic
======================

2020-09-30T14:46:47.146531Z broker.err[0]: rc2.0: /bin/bash -c sh ./t3000-mpi-basic.t  --logfile Exited (rc=1) 6.7s
2020-09-30T14:46:47.566499Z broker.err[0]: cleanup.1: flux-job: Canceled 1 jobs (0 errors)
flux-start: 0 (pid 3285) exited with rc=1
# ./t3000-mpi-basic.t: flux session size will be 
ok 1 - mpi hello singleton
PASS: t3000-mpi-basic.t 1 - mpi hello singleton
not ok 2 - mpi hello all ranks
FAIL: t3000-mpi-basic.t 2 - mpi hello all ranks
#	
#		run_program 5 ${SIZE} ${SIZE} ${FLUX_BUILD_DIR}/t/mpi/hello \
#			| tee allranks.$OPTS \
#			&& grep -q "There are ${SIZE} tasks" allranks.$OPTS
#	
# failed 1 among 2 test(s)
1..2
ERROR: t3000-mpi-basic.t - exited with status 1
```

**Comment by grondo** [comment]
Shall we go ahead and merge this one and tag v0.20.0?

**Comment by garlick** [comment]
> Shall we go ahead and merge this one and tag v0.20.0?

Ooh let's!

**Comment by garlick** [comment]
Hmm, this time ASAN builder failed with
```
##[error]The operation was canceled.
```
Restarting....

File: NEWS.md (hunk 1 above)
Comment (chu11) [reply NEWS.md]: Did you mean "from manager to job-info?" If that's longer than a line,  maybe just "to job-info".

Comment (garlick) [reply NEWS.md]: DOH!

**Pull request merged**